src/repository/screens/repository.screen.js

```diff
@@ -429,15 +420,15 @@ class Repository extends Component {
           >
             {openPulls
               .slice(0, 3)
-              .map(item => (
+              .map(item =>
                 <IssueListItem
                   key={item.id}
                   type="pull"
                   issue={item}
                   navigation={navigation}
                   language={language}
                 />
-              ))}
+              )}
           </SectionList>
         </ParallaxScroll>
 
```

Comment:
it seems that this file also shouldn't be part of the PR, only formatting changes

Reply:
if the file isn't changed, nor included in the commit, you won't get blocked